pom.xml

```diff
@@ -780,9 +780,9 @@
         <activeByDefault>true</activeByDefault>
       </activation>
       <properties>
-        <presto.version>316</presto.version>
+        <presto.version>333</presto.version>
```

Comment:
similar to one of the able comment, can we design this a variable and use everywhere?

Reply:
`<presto.version>` is a variable itself.